src/SymfonyCache/CustomTtlListener.php

```diff
@@ -89,12 +99,13 @@ public function cleanResponse(CacheEvent $e)
             } else {
                 $response->headers->addCacheControlDirective('s-maxage', $smaxage);
             }
+
+            $response->headers->remove(static::SMAXAGE_BACKUP);
```

Comment:
ah, i see. yeah agree to move the removal in here, because clearly we don't need to remove if it is not present :joy: 

but good to keep removing the ttl header without checking for its existence.